lib/itax_code/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ItaxCode
-  VERSION = "2.0.3"
+  VERSION = "2.1.0"
```

Comment:
The versions mustn't be bumped manually, as it's automatically done by the [release-please-action](https://github.com/matteoredz/itax-code/blob/main/.github/workflows/release.yml#L24) upon merging a release PR.

```suggestion
  VERSION = "2.0.3"
```